src/components/dialogs/network-modifications/vsc/converter-station/converter-station-pane.tsx

```diff
@@ -91,11 +91,11 @@ const ConverterStationPane: FunctionComponent<VscConverterStationPaneProps> = ({
             size="small"
             fullWidth
             label={'ID'}
-            value={previousValues?.id || ''}
-            InputProps={{
-                readOnly: true,
-            }}
-            disabled
+            value={previousValues?.id}
+            // InputProps={{
+            //     readOnly: true,
+            // }}
+            // disabled
```

Comment:
if it is not needed, why not just delete it

Reply:
done